File: Python/1280.py
a, b = map(int, input().split())
res = sum([i if i%2!=0 else -i for i in range(a, b + 1)])
print("".join(["+%d" % i if i%2!= 0 else "-%d" % i for i in range(a, b + 1)]) + "=" + str(res))

File: C/1280.c
#include <stdio.h>

int main(){
	int a, b;
	scanf("%d %d", &a, &b);
	int sum = 0;
	
	for(int i = a; i <= b; i++){
		if(i%2!=0){
			sum += i;
			printf("+%d", i);
		} else{
			sum -= i;
			printf("-%d", i);
		}
	}
	
	printf("=%d", sum);
	
	return 0;
}